app/src/pages/embedding/Embedding.tsx

```diff
@@ -424,7 +422,8 @@ const PointCloudDisplay = ({
                   <PointCloudDisplaySettings />
                 </TabPane>
                 <TabPane name="Hyperparameters">
-                  <PointCloudParameterSettings />
+                  <UMAPParameterSettings />
+                  <HDBSCANParameterSettings />
```

Comment:
Did you mean to put this here?

Reply:
please feel free to reorganize. i only made it a separate component for my own testing purposes, and will defer to you regarding UI placements